src/react-imgix.js

```diff
@@ -4,44 +4,20 @@ import ReactDOM from "react-dom";
 import React, { Component } from "react";
 import PropTypes from "prop-types";
 
-import processImage from "./support.js";
-
-// Best way to include an img with an empty src https://stackoverflow.com/a/5775621/515634 and https://stackoverflow.com/a/19126281/515634
-// Using '//:0' doesn't work in IE 11, but using a data-uri works.
-const EMPTY_IMAGE_SRC =
-  "data:image/gif;base64,R0lGODlhAQABAAD/ACwAAAAAAQABAAACADs=";
+import targetWidths from "./targetWidths";
+import constructUrl from "./constructUrl";
 
 const PACKAGE_VERSION = require("../package.json").version;
-
-const roundToNearest = (size, precision) =>
-  precision * Math.ceil(size / precision);
+const NODE_ENV = process.env.NODE_ENV;
```

Comment:
Interesting. How does this end up manifesting when this code is run in-browser?

Reply:
This is an extremely common pattern which is used everywhere in React. It is usually defined by webpack plugin or webpack itself.